frontend/src/metabase/browse/components/utils.tsx

```diff
@@ -102,24 +104,78 @@ export function isDatasetScalar(dataset: Dataset) {
   return dataset.data.cols.length === 1 && dataset.data.rows.length === 1;
 }
 
-export function getDatasetScalarValueForMetric(dataset: Dataset) {
-  const isScalar = isDatasetScalar(dataset);
-  if (!isScalar) {
+export function isDatasetTemporalMetric(dataset: Dataset) {
```

Comment:
Just realized that these utils don't handle query errors. Can we check for `dataset.error` and return `false`?